lib/engines/hcktest/tests.rb

```diff
@@ -132,11 +132,34 @@ def wait_queued_test(id)
       end
     end
 
-    def test_parameters(test_name)
+    def select_test_config(test_name, config)
       @project.engine.config.tests_config
               .select { _1.tests.include?(test_name) }
-              .flat_map(&:parameters)
-              .to_h { |parameter| [parameter.name, parameter.value] }
+              .flat_map(&config)
+    end
+
+    def test_parameters(test_name)
+      select_test_config(test_name, :parameters)
+        .to_h { |parameter| [parameter.name, parameter.value] }
+    end
+
+    def run_guest_command(test, command)
+      @tools.run_on_machine(@client.name, command, command)
+      @tools.run_on_machine(@support.name, command, command) if support_needed?(test)
+    end
+
+    def run_test_commands(test, type)
+      select_test_config(test['name'], type).each do |commands|
+        if commands.guest_run
+          @logger.info("Running command (#{commands.desc}) on clients")
```

Comment:
Better to have log `@logger.info("Running command (#{commands.desc}) on client #{name}")` instead of a generic one. This is more complicated to debug fail.